mlrun/api/db/sqldb/db.py

```diff
@@ -1021,16 +1021,23 @@ def _delete_resources_tags(self, session: Session, project: str):
         for tagged_class in _tagged:
             self._delete(session, tagged_class, project=project)
 
-    def _delete_resources_labels(self, session: Session, project: str):
-        for labeled_class in _labeled:
-            if hasattr(labeled_class, "project"):
-                self._delete(session, labeled_class, project=project)
-
-    def list_functions(self, session, name=None, project=None, tag=None, labels=None):
+    def list_functions(
+        self,
+        session: Session,
+        name: str = None,
+        project: str = None,
+        tag: str = None,
+        hash_key: str = None,
+        labels: List[str] = None,
```

Comment:
note that the sqldb and the fileDB both inherit from the api/db/base, while the order of the arguments in each of them is different. `FileDB` and `Base` have the `hash_key` as the last argument and in the sqldb the `labels` is the last. this is very prone to bugs.